### ISSUE TITLE
broken global argument handling with Dockerfile.base

### ISSUE THREAD

**Issue by andreasvoigt** [rudxde/lerna-dockerize#350]
**Describe the bug**
Docker has a specific way to handle globally defined ARGs (ARGs above a FROM stage). Those globally defined ARGs can be reused in several FROM blocks. E.g. if you want to use ARGs which are defined in the `Dockerfile.base`, you can reuse them in the package specific Dockerfiles via redefinition. But those definitions will be eleminated via lerna-dockerize.

***Example `Dockerfile.base`*** 
```
# This is the base stage, which prepares the project for further builds with lerna-dockerize.

# global arguments
ARG AWESOME_ARG=test

FROM node:12-slim
ARG AWESOME_ARG

RUN echo "varibale content: $AWESOME_ARG"
# do other stuff
```

**To Reproduce**
1. add a block with a ARG definition above the FROM command in within the Dockerfile.base
2. run `npm run lerna-dockerize` in you project
3. see the compiled output --> your defined ARG does not exist in the output file

**Expected behavior**
The globally defined ARG should be taken into the output file.

**Desktop (please complete the following information):**
 - OS: Linux

**Additional context**
See further information about ARG handling in docker documentation: https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact

**Comment by rudxde** [comment]
Hi @andreasvoigt ,
can you check if the issue is resolved in the current build version?

```
npm i --save-dev lerna-dockerize@build
```

**Comment by andreasvoigt** [comment]
Good morning!
The patch works. Thank you very much!

**Comment by rudxde** [comment]
Will create an patch release soon

**Comment by rudxde** [comment]
The fix is part of the release [0.9.3](https://github.com/rudxde/lerna-dockerize/releases/tag/v0.9.3)
